.gitlab-ci.yml

```diff
@@ -56,18 +56,28 @@ validate-log-integrations:
 trigger-agent-build:
   stage: trigger
   image: $VALIDATE_AGENT_BUILD
+  # Exclude release branches without dependency change
+  rules:
+    - if: ($CI_COMMIT_BRANCH =~ /^7.[0-9]{2}.x/)
+      changes:
+        - datadog_checks_base/datadog_checks/base/data/agent_requirements.in
+      when: always
+    - if: ($CI_COMMIT_BRANCH !~ /^7.[0-9]{2}.x/)
+      when: always
   cache:
     <<: *slack-cache
   script:
     - export SLACK_CACHE_DIR="${PWD}/.slack-cache"
     - DATADOG_AGENT_PIPELINE_URL=$(aws ssm get-parameter --region us-east-1 --name ci.integrations-core.datadog-agent-pipeline-url --with-decryption --query "Parameter.Value" --out text)
     - export AGENT_REQUIREMENTS_CHANGED="false"
     - git fetch origin master
-    - git --no-pager diff --exit-code --name-only HEAD $(git merge-base HEAD FETCH_HEAD) -- datadog_checks_base/datadog_checks/base/data/agent_requirements.in || export AGENT_REQUIREMENTS_CHANGED="true" || true
+    - BASE_REF=$(git merge-base HEAD FETCH_HEAD)
+    - git --no-pager diff --exit-code --name-only HEAD $BASE_REF -- datadog_checks_base/datadog_checks/base/data/agent_requirements.in || export AGENT_REQUIREMENTS_CHANGED="true" || true
     - |
       if [[ "$AGENT_REQUIREMENTS_CHANGED" = "true" ]]; then
+          export RELEASE_BRANCH=$(git tag -l '7\.*' --points-at $BASE_REF | cat | sed 's/\..-.*/\.x/')
```

Comment:
Shouldn't the change just be this?

Reply:
Ah wait looks like there were just other changes from the `.gitlab-ci.yml` file that this branch didn't have.